app/api/helpers/files.py

```diff
@@ -78,13 +79,17 @@ def create_save_resized_image(image_file, basewidth=None, maintain_aspect=None,
         return None
 
     filename = '{filename}.{ext}'.format(filename=get_file_name(), ext=ext)
-    data = urllib.request.urlopen(image_file).read()
+    try:
+        data = urllib.request.urlopen(image_file).read()
+    except urllib.error.HTTPError:
+        raise UnprocessableEntity(
+            {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
```

Comment:
This is a helper file. It can't be so specific. We don't know if the attribute will even contain a `original-image-url`. Please raise a HTTPError and add this specific try except block where we are calling this function.

Reply:
> We don't know if the attribute will even contain a original-image-url

We are building `original_image_url` `large_image_url` `thumbnail_image_url` & `icon_image_url`  from `data['original_image_url']` just resizing the same image accordingly.
See https://github.com/fossasia/open-event-server/blob/development/app/api/events.py#LC397
So, it should be kept in helper function only.